### DISCUSSION TITLE
2024-09-24 온라인 회의

### DISCUSSION THREAD

**Discussion by jimin52** [jiphyeonjeon-42/backend-nest#61] [General]
## 아젠다
- 진행상황 공유
- 이슈 공유
- 다음 계획

**Comment by scarf005** [comment]
## 진행상황

- #60

## 이슈

- 공용으로 쓰이는 zod schema 추출해야 함

## 다음 계획

- 프로젝트 설정 등 문서화

**Comment by nyj001012** [comment]
## 진행상황 공유
- #48
 - 실행 확인 후 PR 오픈할 예정

## 이슈 공유
- 더이상 진행이 불가능할 듯
- 취업준비
